src/Native/Runtime/GCMemoryHelpers.inl

```diff
@@ -7,6 +7,18 @@
 // Unmanaged GC memory helpers
 //
 
+#if defined(HOST_64BIT)
+static const int card_byte_shift = 11;
+static const int card_bundle_byte_shift = 21;
+#else
+static const int card_byte_shift = 10;
+
+#ifdef FEATURE_MANUALLY_MANAGED_CARD_BUNDLES
+#error Manually managed card bundles are currently only implemented for AMD64.
+#endif
```

Comment:
Should this be outside `#if defined(HOST_64BIT) / #else / #endif` ?

Reply:
This code block was directly taken from the runtime code. I believe that the error message is outdated and changed it.